tests/pipeline/resources/read-from-component/pipeline.yaml

```diff
@@ -0,0 +1,26 @@
+- type: producer
+  name: producer
+  to:
+    topics:
+      ${output_topic_name}:
+        type: output
+
+- type: streams-app
+  name: consumer1
+
+- type: streams-app
+  name: consumer2
+  from:
+    components:
+      - ${pipeline_name}-producer
```

Comment:
This should work without pipeline_name as we can only reference components within our pipeline

Reply:
it does if the other component has `prefix: ""`

Reply:
prefix definition for reference https://github.com/bakdata/kpops/blob/3a4fac55d2e5e0ddffd02a216aad9fda82c90b9f/kpops/components/base_components/pipeline_component.py#L30